packages/marketplace/src/sagas/client.ts

```diff
@@ -29,36 +29,19 @@ export const clientDataFetch = function*({ data }) {
     const isFilteringForDirectApiApps = category === 'DIRECT_API_APPS_FILTER'
 
     const [apps, featuredApps, categories] = yield all([
-      call(fetcher, {
-        url: `${URLS.apps}?${setQueryParams({
-          clientId,
-          category: isFilteringForDirectApiApps ? undefined : category,
-          [searchBy]: search,
-          pageNumber: page,
-          pageSize: APPS_PER_PAGE,
-          IsFeatured: isFilteringForDirectApiApps ? undefined : false,
-          IsDirectApi: isFilteringForDirectApiApps ? true : undefined,
-        })}`,
-        api: window.reapit.config.marketplaceApiUrl,
-        method: 'GET',
-        headers: generateHeader(window.reapit.config.marketplaceApiKey),
+      call(fetchAppsList, {
+        clientId,
+        category: isFilteringForDirectApiApps ? undefined : category,
+        [searchBy]: search,
```

Comment:
could we change to `searchBy` ?

Reply:
Sorry what do you mean?
Here, `searchBy` is key, and `search` is value

Reply:
is searchBy variables ?

Reply:
yes

Reply:
`[searchBy]: search` can be translated into `appName: 'app'` or `developerId: 'id1'`, etc...

Reply:
ah it's ok